### DISCUSSION TITLE
Encounter Error message when launching stable diffusion automatic 1111

### DISCUSSION THREAD

**Discussion by Jerry20221227** [OpenTalker/SadTalker#823] [Q&A]
After i upgarded stable diffusion automatic 1111 to version 1.80, launch the extension of SadTalker encounter error. Anyone know how to solve it?

**Comment by oisilener1982** [comment]
revert to 1.7 and everything will be fine

**Comment by jmr022024** [comment]
SadTalker works ok with version:             v1.8.0  •  python: 3.10.13  •  torch: 2.0.1+cu118  •  xformers: 0.0.20
SadTalker does NOT work with version:   v1.8.0  •  python: 3.10.13  •  torch: 2.1.2+cu121  •  xformers: 0.0.23.post1
